openassessment/xblock/static/js/src/lms/oa_staff_info.js

```diff
@@ -58,7 +58,41 @@ OpenAssessment.StaffInfoView.prototype = {
                 view.showLoadError('student_info');
             });
     },
+    
+    /**
+     Upon request, submits an override grade.
+     **/
+    peerScoreOverride: function() {
+    	var sel = $('#openassessment__staff-info', this.element);
+    	var student_id = sel.find('#openassessment__student_id').val();
+    	var override_score = sel.find('#openassessment_override_score').val();
+    	var points_possible = sel.find('#openassessment_points_possible').html();
+    	this.server.peerScoreOverride(student_id, points_possible, override_score).done(
+            function(override_score) {
+                $('#openassessment_points_override', this.element).html(override_score);
+            }
+        ).fail(function(errMsg) {
+                $('#openassessment_points_override', this.element).html(errMsg);
+        });
+    },
+    
+    ScoreOverrideHandler: function() {
+        var sel = $('#openassessment__student-info', this.element);
+        var view = this;
 
+        if (sel.length <= 0) {
+            return;
+        }
+        
+        // Click handler for overriding a student's peer score.
+        sel.find('#submit_override_score').live('click',
```

Comment:
Any reason this doesn't use the same `.click` call as line 142, instead of `live`? It seems that `live` is deprecated.